---
title: Multi-metagenomics samples analysis
keywords: sample homepage
tags: [getting_started]
sidebar: mydoc_sidebar
permalink: index.html
summary: This document will give a step by step instruction to analyze a group of envrionmental metagenomics samples using metagenomics profile (MetaP v0.1).
---
Motivation:

As the decline of sequencing price, generating tens or even hundreds of metagenomcis samples in one study become regular configuration for metagenomics wide association studies, this manual shows how to process multi-metagenomic samples from one type of environment to 1) generate a gene catalog from this environment and evaluate the completeness of the catalog; 2) use the gene co-abundance information to extrate the so-called metagenomics species from samples(the underline assumption is that genes in one species/strain/genome tend to have the same abundance variation across samples); 3) assess the completeness and contamination of the reconstructed metagenomic geneomes. We apply the activated sludge system as demonstration.      

## Clone MetaP to local computer

First download or clone the source code from the [Github repo](https://github.com/tomjohnson1492/documentation-theme-jekyll).

## Prepare your multi-metagenomics samples


## Quality control

```
open Gemfile
```

{% include links.html %}
